firestore.rules

```diff
@@ -16,22 +22,30 @@ service cloud.firestore {
             	return get(/databases/$(database)/documents/boards/$(id)).data.owner == request.auth.uid || (isMember() && get(/databases/$(database)/documents/boards/$(id)/members/$(request.auth.uid)).data.admin == true)
             }
 
-            allow get: if isMember()
+            allow get: if isMember() || isAdmin()
             allow list: if false
             allow create: if request.auth.uid != null && request.auth.uid == request.resource.data.owner
             allow update, delete: if isAdmin()
 
-            match /applicants/{uid} {
+            match /pending/{uid} {
                 allow read: if isAdmin()
                 allow create: if isAdmin() || request.auth.uid == uid
                 allow update: if isAdmin()
                 allow delete: if isAdmin() || request.auth.uid == uid
             }
 
             match /members/{uid} {
```

Comment:
Do you think we should check the vote limits on the DB side or just leave this logic to the clients? 
These checks (and the necessary code) do have a negative effect on performance and costs. It would be much easier to use the Firestore functions 'increment' and 'decrement' instead and handle everything on the clients.

Reply:
Yea, it would be a nice approach! 👍